### PR TITLE
Add documentation on S3 imports

### PR DESCRIPTION
**Pull request by matkuliak**
## Checklist

 - [ ] Link to issue this PR refers to (if applicable): 
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by amotl** [comment]
Hi Matej,

thanks for this patch.

I have a few suggestions about the screenshot images. Recently, I spent a few minutes on shrinking the screenshot file sizes, because I think 2.5 MB is way too large, and introduces recognizable delays even on todays Internet's fast pipes.

I usually use something around like this, but I will be happy to learn more.
```
convert -resize 1024x -quality 100 cluster-import-tab-choice.png test.png
oxipng test.png
```

Other than this, I think reading flow and information conveyance can be tremendously improved by "zooming in to the feature at hand". As a reader, you can't recognize anything detailed anyway when looking at the _full screenshot_. Or is it just my bad eyes?

On behalf a recent article, where I've demonstrated the Cloud Import feature, I just "zoomed in" to the import dialag, and I think it is much better like this.

https://github.com/crate/ml-sandbox/blob/main/timeseries-data/article/01-about-intro.md#importing-data

With kind regards,
Andreas.

File: docs/reference/overview.rst [hunk 1]
```diff
@@ -315,6 +315,41 @@ Gzip compressed files are also supported.
 .. image:: ../_assets/img/cluster-import-tab-url.png
    :alt: Cloud Console cluster upload from URL
 
+.. _overview-cluster-import-s3:
+
+Import from private S3 bucket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CrateDB Cloud allows convenient imports directly from a S3 storage.
```
Comment (SStorm) [inline docs/reference/overview.rst:323]:
```suggestion
CrateDB Cloud allows convenient imports directly from S3-compatible storage. 
```

File: docs/reference/overview.rst [hunk 2]
```diff
@@ -315,6 +315,41 @@ Gzip compressed files are also supported.
...
    :alt: Cloud Console cluster upload from URL
 
+.. _overview-cluster-import-s3:
+
+Import from private S3 bucket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CrateDB Cloud allows convenient imports directly from a S3 storage. 
+To import a file form bucket, provide the name of your bucket, and path to
+file. The S3 Access Key ID, and S3 Secret Access Key are also needed. You can
```
Comment (SStorm) [inline docs/reference/overview.rst:325]:
```suggestion
the file. The S3 Access Key ID, and S3 Secret Access Key are also needed. You can
```

File: docs/reference/overview.rst [hunk 3]
```diff
@@ -315,6 +315,41 @@ Gzip compressed files are also supported.
...
+
+       {
+        "Version": "2012-10-17",
+        "Statement": [
+        {
+            "Sid": "AllowGetObject",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "*"
+            },
```
Comment (SStorm) [inline docs/reference/overview.rst:347]: Maybe remove the principal, it's not really needed. 
```suggestion
```

**Review by SStorm — APPROVED** [review]
A few minor points below :100:

File: docs/reference/overview.rst (hunk 3 above)
Comment (matkuliak) [reply docs/reference/overview.rst:347]: As discussed, it was needed in my last test so let's keep it. Thanks!

**Comment by matkuliak** [comment]
> Hi Matej,
> 
> I have a few suggestions about the screenshot images

Hi @amotl. Thank you for the suggestions. You're right, the sizes could be improved. I applied oxipng on all the pictures for an average 75% size reduction, so now coming in at around 0.5MB. I will do this for all the pictures in a separate PR and add it to my workflow.

As for the Magick I'll go without it for now, hope that's okay. 
Zooming in on the pictures is for a debate, I considered this a couple of times previously. The issue, for me, is that in some cases it takes away some of the important context. E.g. you're not sure where to look for the specific field in the screenshot. And applying this case by case makes for a significantly worse visual of document, so I opted for universal fullscreen screenshots until now.

**Pull request merged**